README.md

```diff
@@ -1,6 +1,8 @@
 # Telicent Design System (DS)
 
-A component library to help developers create frontend applications for Telicent platform apps. Visit the [Storybook](https://telicent-oss.github.io/ds/) to learn more.
+A component library to help developers create frontend applications for [Telicent.io](https://telicent.io/) platform apps.
+
+Visit the [Storybook](https://telicent-oss.github.io/ds/) to learn more.
```

Comment:
I wasn't sure if this is a broken link or just didn't work because I haven't installed the @telicent-oss/ds package NPM package?

However, I did find the local installation of StoryBook worked fine by following the instructions in the 'Run Storybook locally' section, below.

Reply:
Yeha - unfortunately it won't work until: https://github.com/Telicent-io/telicent-ds/issues/377

Reply:
Ah, that makes sense if the endpoint isn't in the repository no wonder it can't be reached.  I'm happy this isn't an issue at this point.